x-pack/plugins/monitoring/public/components/kuery_bar/suggestion_item.tsx

```diff
@@ -44,19 +44,19 @@ const suggestionItemContainerStyle = (isSelected?: boolean) => (theme: UseEuiThe
   css`
     display: flex;
     flex-direction: row;
-    font-size: ${theme.eui.euiFontSizeS};
-    height: ${theme.eui.euiSizeXL};
+    font-size: ${theme.euiTheme.font.scale.s};
+    height: ${theme.euiTheme.size.xl};
     white-space: nowrap;
-    background-color: ${isSelected ? theme.eui.euiColorLightestShade : 'transparent'};
+    background-color: ${isSelected ? theme.euiTheme.colors.lightestShade : 'transparent'};
   `;
 
 const suggestionItemFieldStyle = (theme: UseEuiTheme) => css`
   align-items: center;
   cursor: pointer;
   display: flex;
   flex-direction: row;
-  height: ${theme.eui.euiSizeXL};
-  padding: ${theme.eui.euiSizeXS};
+  height: ${theme.euiTheme.size.xl};
```

Comment:
I agree that destructuring is fun and makes the code cleaner but I'd love to keep the changes to a minimum and what's necessary for this PR. I hope that's fine for you 😄 

Thanks for the awesome suggestions though!

Reply:
Imho, this will not add more changes. You're touching the code (or in parts adding the code) and changing `theme.euiTheme` to `euiTheme` is a cleanup that doesn't actually change much besides making the code a bit leaner. 

Since we're here, let's make that change, since otherwise it likely won't happen.